docs/source/frequently_asked_questions.ipynb

```diff
@@ -419,6 +419,21 @@
     "feature_matrix[[\"COUNT(sessions WHERE product_id_device = 5 and tablet)\"]]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Can I create an `EntitySet` using Dask dataframes? (BETA)\n",
+    "\n",
+    "Support for Dask EntitySets is still in Beta - if you encounter any errors using this approach, please let us know by creating a [new issue on Github](https://github.com/FeatureLabs/featuretools/issues).\n",
+    "\n",
+    "Yes! Featuretools supports creating an `EntitySet` from Dask dataframes. You can simply follow the same process you would when creating an `EntitySet` from pandas dataframes.\n",
```

Comment:
I notice we are capitalizing Dask mostly but not Pandas

Reply:
That was done because that is typically how they are styled in their respective docs.
[https://pandas.pydata.org/](https://pandas.pydata.org/)
[https://dask.org/](https://dask.org/)